src/pages/users/[id]/share.tsx

```diff
@@ -148,13 +152,21 @@ const SharePage = () => {
         throw new Error("Failed to share information");
       }
     } catch (error) {
-      alert("An error occurred while sending the message. Please try again.");
+      toast.error(
+        "An error occurred while sending the message. Please try again."
+      );
       return;
     }
 
-    // Updates local storage with new private note and timestamp
-    updateUserFromOutboundTap(user.encPk, privateNote);
-    alert(`Successfully shared information with ${user.name}!`);
+    // Updates local storage and activity feed
+    try {
+      await loadMessages({ forceRefresh: false });
```

Comment:
Not sure we should be loading on the share screen, it will make the share interaction feel super slow. Would prefer we have some sort of spinner on the social tab when we redirect there